python_modules/dagster/dagster/core/definitions/repository_definition.py

```diff
@@ -1164,3 +1206,16 @@ def source_assets_by_key(self) -> Dict[AssetKey, SourceAsset]:
     # overwritten. Therefore, we want to maintain the call-ability of repository definitions.
     def __call__(self, *args, **kwargs):
         return self
+
+
+def is_resource_dict(defn: Any) -> bool:
```

Comment:
I think we prefer `object` to `Any` in these cases, because it enforces that we can't assume anything about the object until we've verified its type.